### ISSUE TITLE
Review CSS & Styling

### ISSUE THREAD

**Issue by thebestsophist** [covid19-guide/project-tracking#8]
# Description
Tina has done the design, it needs to be reviewed

# Slack Channel
covid-guide

# Supporting docs

**Comment by ryanpitts** [comment]
Best place to review is https://covid19-guide.herokuapp.com/ ?

**Comment by knowtheory** [comment]
Yes until https://github.com/covid19-guide/project-tracking/issues/10 is resolved.

**Comment by knowtheory** [comment]
Since the logo is an SVG with a transparent background, we may need to produce one w/ a solid white fill color, for the times when the logo shows up on a non-white background.

**Comment by tinabeans** [comment]
I updated the logo in the article header to a mini version of the logo (same as favicon design) that's all white!

**Comment by knowtheory** [comment]
a nubbin'
![Screen Shot 2020-03-18 at 2 58 08 PM](https://user-images.githubusercontent.com/192/77011391-049a0600-6929-11ea-8935-2853de2cb465.png)
![Screen Shot 2020-03-18 at 2 58 17 PM](https://user-images.githubusercontent.com/192/77011401-0794f680-6929-11ea-988b-e5dc634643d0.png)

**Comment by tinabeans** [comment]
Pushed de-nubbin-ifinication css~
